clean_project/search_cli.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Search a word in a file and return all the occurrence specified the line numbers
+"""
+import argparse
+import cmd
+import json
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+class Search(cmd.Cmd):
+
+    def __init__(self, json_path):
+        cmd.Cmd.__init__(self)
+        self.json_path = json_path
+
+    def do_search(self, search_word):
```

Comment:
HOLD ON: why is this function the exact replica of the one in search.py? if you change the logic there, you'd have to duplicate the changes here too. That's not good AT ALL! find a way to abstract the logic and call the `search()` function from here